src/Library/demos/Power Profile Monitor/main.py

```diff
@@ -0,0 +1,27 @@
+import gi
+
+gi.require_version("Adw", "1")
+from gi.repository import Adw, Gio
+import workbench
+
+power_profile_monitor = Gio.PowerProfileMonitor.dup_default()
+overlay = workbench.builder.get_object("overlay")
+
+
+def on_power_saver_enabled(monitor, enabled):
```

Comment:
Since they are not used just use "*args", to avoid confusion

Reply:
Oh wait right, I forgot about the earlier discussion already, my bad! I think better would be what I proposed above: Just use the arguments. So use "enabled" and add an underscore to "monitor" since it's not used.